File: Level_1/Term_2/Progrmming 2/Ramadan-Questions/Day_4/The solution/Python/Q_1.py
grade = -1  # هنا لازم نديله قيمة بتحقق شرط اللوب علشان يدخل فيه اول مرة او استعمل دو  وايل لو مش عايز ادي قيمة اولية
# Read grade from user if <= 100 and >= 0
while grade > 100 or grade < 0:  # اللوب دي بجبر اليوزر انو يدخل درجة موجبة و مش اكبر من 100
    grade = int((int(input("Please enter your grade: ")) / 10))
# Check grade using switch statement only
# Python does not contain a switch clause, so we will use the if and elif statement >.<
if grade == 10 or grade == 9:
    print("Excellence n_n")
elif grade == 8:
    print("Very good u_u")
elif grade == 7:
    print("Good h_h")
elif grade == 6:
    print("Acceptable @_@")
else:
    print("Not so lucky >_<, try again ^_^")
# Programming is very, very fun 🖤

File: Level_1/Term_2/Progrmming 2/Ramadan-Questions/Day_2/The solution/CPP/Q_1.cpp
#include <iostream>
using namespace std;

int main()
{
    int grade;

    // Read grade from user
    cout << "Please enter your grade: ";
    cin >> grade;

    // Check grade using if and else if and else only
    if(grade >= 90)
        cout << "Excellence n_n" << endl;
    else if(grade >= 80)
        cout << "Very good u_u" << endl;
    else if(grade >= 70)
        cout << "Good h_h" << endl;
    else if(grade >= 60)
        cout << "Acceptable @_@" << endl;
    else
        cout << "Not so lucky >_<, try again ^_^" << endl;
    return 0;
}
